frontend/next.config.js

```diff
@@ -1,5 +1,11 @@
+const WindiCSS = require("windicss-webpack-plugin").default;
+
 module.exports = {
-    future: {
-        webpack5: true,
-    },
-}
\ No newline at end of file
+  webpack: (config) => {
+    config.plugins.push(new WindiCSS());
```

Comment:
pls

Reply:
Just let it be, you'll love it, everyone is using Tailwind nowadays 🌈

Reply:
Lol I don't follow trends homie. Sound engineering management means introducing less fad technologies and sticking with what people know.